### PR TITLE
update readme to remove insiders specific references + some cleanup

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
For #8548

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,21 +36,21 @@ To set it up, follow the steps [here](https://github.com/microsoft/vscode-jupyte
...
 
--   **Step 1.** Install [VS Code Insiders](https://code.visualstudio.com/insiders/)
+-   **Step 1.** Install [VS Code](https://code.visualstudio.com/)
 
 -   **Step 2** If not working with Python, make sure to have a Jupyter kernelspec that corresponds to the language you would like to use installed on your machine.
 
 -   **Step 3.** Install the [Jupyter Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
 
--   **Step 4.** Open or create a notebook file and start coding!
+-   **Step 4.** Open or create a notebook file and start coding.
```
Comment (IanMatthewHuff) [inline README.md:49]: I removed basically all !s out of a strong personal preference ;). However I'm totally willing to put them back in if I'm outvoted here.

**Review by rchiodo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ The extension is available in multiple languages: `de`, `en`, `es`, `fa`, `fr`,
 
     -   If someone has already [filed an issue](https://github.com/Microsoft/vscode-jupyter) that encompasses your feedback, please leave a 👍/👎 reaction on the issue.
 
--   Contributions are always welcome! Please see our [contributing guide](https://github.com/Microsoft/vscode-jupyter/blob/main/CONTRIBUTING.md) for more details.
+-   Contributions are always welcome. Please see our [contributing guide](https://github.com/Microsoft/vscode-jupyter/blob/main/CONTRIBUTING.md) for more details.
```
Comment (greazer) [inline README.md:99]: Seems this could read as "Contributions are always welcome, so please see our..."

File: README.md [hunk 3]
```diff
@@ -79,11 +79,11 @@ To see all available Jupyter Notebook commands, open the Command Palette and typ
 
 Learn more about the rich features of the Jupyter extension:
 
--   [IntelliSense](https://code.visualstudio.com/docs/python/editing#_autocomplete-and-intellisense): Edit your code with auto-completion, code navigation, syntax checking and more!
+-   [IntelliSense](https://code.visualstudio.com/docs/python/editing#_autocomplete-and-intellisense): Edit your code with auto-completion, code navigation, syntax checking and more.
 
     -   _May be limited due to kernelspec of choice_
```
Comment (greazer) [inline README.md:84]: This disclaimer seems unnecessary, even though it's true. I mean intellisense is also limited to the language you're working with and we're not disclaiming that.

**Review by greazer — COMMENTED** [review]
Thanks for doing this Ian!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!... !!!!!!!! !   !            !

**Pull request merged**